pkg/oc/cli/cmd/status.go

```diff
@@ -86,7 +86,10 @@ func NewCmdStatus(name, baseCLIName, fullName string, f *clientcmd.Factory, out
 	}
 
 	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", opts.outputFormat, "Output format. One of: dot.")
-	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", opts.verbose, "See details for resolving issues.")
+	cmd.Flags().BoolVarP(&opts.suggest, "verbose", "v", opts.suggest, "See details for resolving issues.")
+	cmd.Flags().MarkDeprecated("verbose", "Use --suggest instead.")
+	cmd.Flags().MarkHidden("verbose")
+	cmd.Flags().BoolVarP(&opts.suggest, "suggest", "s", opts.suggest, "See details for resolving issues.")
```

Comment:
no shorthand.  They are scarce

Reply:
why not just drop the shorthand and use "verbose"?

Reply:
This is not verbosity level in any way, this is a boolean flag whether to show suggestions. That's why this better reflects the intent.